packages/block-library/src/template-part/edit/index.js

```diff
@@ -59,8 +62,9 @@ export default function TemplatePartEdit( {
 
 	if ( postId ) {
 		// Part of a template file, post ID already resolved.
+		const BlockWrapper = Block[ tagName ];
 		return (
-			<>
+			<BlockWrapper>
```

Comment:
We may need to add a wrapper like this to the placeholder as well.

Reply:
Yeah, we could do that. I didn't think it mattered in the place holder state :)